client/my-sites/plans/jetpack-plans/selector.tsx

```diff
@@ -177,6 +177,9 @@ const SelectorPage: React.FC< SelectorPageProps > = ( {
 		( variation: Iterations | null ) => `jetpack-plans__iteration--${ variation ?? 'default' }`
 	);
 
+	const urlParams = new URLSearchParams( window.location.search );
+	const currencyCode = urlParams.get( 'currency' );
```

Comment:
For simplicity as well as consistency, I think it would be better to use the `urlQueryArgs` prop that already exists for getting the `currency` arg from the url.

```suggestion
	const { currency: currencyCode } = urlQueryArgs;
```

Reply:
I somehow did not see this in all my research 😆 thank you!

Reply:
Haha, awesome, code looks all good 👍